### DISCUSSION TITLE
what is the proper way to config args useContractRead

### DISCUSSION THREAD

**Discussion by superposition** [wevm/wagmi#926] [Q&A]
Is there any insight why this configuration is incorrect?

```  
const { data, isError, isLoading, error } = useContractRead({
    addressOrName: contractAddress,
    contractInterface: CryptosStaking.abi,
    functionName: "getEpy",
    overrides: {
      from: address,
      value: ethers.utils.parseEther("10000"),
    },
  });
```

<img width="541" alt="image" src="https://user-images.githubusercontent.com/3332969/189053831-70552375-ae78-4e16-ade4-8437fc379184.png">
 
https://goerli.etherscan.io/address/0x7A07dbCe902FA99B8BD3B0db2bD9C6e6F3b12Ed8#readContract

**Comment by dev-cz** [comment]
have you tried removing the override and replacing it with 

args: ethers.utils.parseEther("10000"),

**Reply by superposition** [reply]
yes, but to no avail
